docs/releases/5.1.txt

```diff
@@ -32,7 +32,9 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* When adding a new user or changing a password in the admin UI, the
+  administrator can chose to set an unusable password effectively disabling
+  a user from logging in with a password.
```

Comment:
I think this would be best placed under the `django.contrib.auth` subsection. Also, I'd suggest a rewording as per:
```suggestion
* Improved password management in the User model now allows admins to enable or
  disable password-based authentication by setting a usable or unusable        
  password, applicable in user creation and change forms.                      
```